python/ccxt/base/exchange.py

```diff
@@ -71,7 +71,7 @@
 import re
 from requests import Session
 from requests.utils import default_user_agent
-from requests.exceptions import HTTPError, Timeout, TooManyRedirects, RequestException
+from requests.exceptions import HTTPError, Timeout, TooManyRedirects, RequestException, ConnectionError as requestsConnectionError
```

Comment:
`requests.exceptions.ConnectionError` shadows the built-in `ConnectionError`, hence the alias.